tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -1019,4 +1072,4 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
         telemetry.send_event('mo', 'conversion_result', 'fail')
         telemetry.end_session('mo')
         telemetry.force_shutdown(1.0)
-        raise e.with_traceback(None)
+        raise e#.with_traceback(None)
```

Comment:
Don't forget to uncomment it when PR is ready.

Reply:
thanks, return back.
BTW, is there a better solution for getting an error stack trace?

Reply:
We actually remove stack trace here to make error message smaller. Otherwise user sees a big stack stare with inner MO files which is not very nice.

Reply:
I mean in development purposes, not as part of the product (it is not a change request or something else, just some hints for me in the future :)), so it is the only known solution, it is ok.

Reply:
Yes, for development purposes commenting this line is the only way to obtain stack trace now. 
Maybe we could make a better approach. Using some env variable or if loglevel=DBUG is set. @rkazants what do you think?